stylesheets/design-patterns/_media-player.scss

```diff
@@ -150,25 +149,48 @@
         &:hover,
         &:focus {
           color: $light-blue;
+          background-color: transparent;
+          outline: none;
+
+          &:before {
+            background-color: $focus-colour;
+          }
+        }
+
+        &:before {
```

Comment:
@fofr looks like the player never worked in IE8... at least according to BrowserStack, all I get is JS errors on the [original link](http://govuk-component-guide.herokuapp.com/components/govspeak/fixtures/with_youtube_embed/preview). Looks fine visually (with this fix), but none of the functionality works.